CHANGELOG.md

```diff
@@ -2,10 +2,11 @@
 
 ### Unreleased
 - [#144](https://github.com/smile-io/ember-polaris/pull/144) [UPDATE] Shopify Polaris `v2.2.0`
-- [#146](https://github.com/smile-io/ember-polaris/pull/146) Add `new` status to the Badge component
+- [#146](https://github.com/smile-io/ember-polaris/pull/146) [UPDATE] Add `new` status to the Badge component
 - [#148](https://github.com/smile-io/ember-polaris/pull/148) [FEATURE] Add `polaris-label` and `polaris-labelled` components.
 - [#151](https://github.com/smile-io/ember-polaris/pull/151) [FIX] Add missing `hidden` class to `polaris-label`.
-- [#152](https://github.com/smile-io/ember-polaris/pull/152) Accept card title as a string or a component
+- [#152](https://github.com/smile-io/ember-polaris/pull/152) [UPDATE] Accept card title as a string or a component
```

Comment:
Is this an update or an enhancement? 🤔 

🤷‍♂️

Reply:
not sure if it's defined anywhere what's an update or enhancement

Reply:
No, not really. To me an enhancement is maybe every so slightly more specifically about improving existing functionality while an update is more general... with that in mind, it all depends on the wording used in the PR summery as to whether it's an enhancement or an update. Doesn't really matter either way 😛